### PR TITLE
Add detail on securing vpn gateway

### PR DESCRIPTION
**Pull request by unman**
Incorporate discussion from https://github.com/QubesOS/qubes-issues/issues/1941

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
@marmarek

File: configuration/vpn.md [hunk 1]
```diff
@@ -16,43 +16,108 @@ Setting up a VPN connection is really not Qubes specific and is documented in yo
...
-One of the best thing in Qubes is that you can use a special type of VM called a ProxyVM (or FirewallVM). The special thing is that your AppVMs see this as a NetVM, and your NetVMs see it as an AppVM. Because of this, you can place a ProxyVM between your AppVMs and your NetVM. This is how the default FirewallVM functions.
+You can also use a type of VM called a ProxyVM [needs Qubes 3.1-rc2 (or later)](https://github.com/QubesOS/qubes-issues/issues/1052). You can place a ProxyVM between your qubes and sys-net, (like the default FirewallVM). We recommend using a dedicated qube for this purpose.
 
 Using a ProxyVM to set up a VPN client gives you the ability to:
 
 -   Separate your VPN credentials from Your NetVM
--   Separate your VPN credentials from Your AppVM data.
--   Easily control which of your AppVMs are connected to your VPN by simply setting it as a NetVM of the desired AppVM.
+-   Separate your VPN credentials from your other qubes.
+-   Easily control which of your qubes are connected to the VP. You do this by setting the Proxy as the NetVM of the desired qube.
```
Comment (marmarek) [inline configuration/vpn.md]: s/VP/VPN/

**Comment by marmarek** [comment]
It looks like very similar change is already done in #147

**Comment by andrewdavidwong** [comment]
Would it be a good idea to attempt to resolve conflicts manually (incl. substantive content inclusion/exclusion/merge decisions) so that we can merge this? @unman, since it's your PR, how would you prefer to proceed?

**Comment by unman** [comment]
@andrewdavidwong  So the change in #147 incorporates *some* of the material that we have been discussing in https://github.com/QubesOS/qubes-issues/issues/1941, but doesn't address the major issues there.
I'm happy to  discuss further in https://github.com/QubesOS/qubes-issues/issues/1941, edit the new content, and submit new PR.

**Pull request closed without merging**